devops/setup.android.env.sh

```diff
@@ -69,7 +69,7 @@ download_and_unzip_dependencies_for_all_architectures(){
     #TODO Get dependencies in more optimized way
     pushd ${ANDROID_BUILD_FOLDER}
         if [ ! -d "indy-android-dependencies" ] ; then
-            git clone https://github.com/evernym/indy-android-dependencies.git
+            git clone https://github.com/sovrin/indy-android-dependencies.git
```

Comment:
~~`sovrin`~~ -> `sovrin-foundation`

Reply:
duplicate of mike's above.

Reply:
This is actually a bigger problem since sovrin does not have that same repo. Could @mikelodder7 or @stevengubler fork this repo to sovrin-foundation github organization?